packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -285,4 +285,26 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /patient/:id
+ *
+ * Returns a patient corresponding to the specified facility at the customer's organization.
+ * The patient is returned with the whole patient model.
+ *
+ * @param   req.cxId      The customer ID.
+ * @param   req.param.id  The ID of the patient to be returned.
+ * @return  The customer's patients associated with the given facility.
+ */
+router.get(
```

Comment:
Just confirming, couldn't we use the regular `GET /patient` b/c of token/access?

Reply:
Yea exactly, that route has a dto that removes all that info